src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -200,7 +200,7 @@ internal virtual CSharpCompilation DeclaringCompilation
                 }
 
                 var sourceModuleSymbol = this.ContainingModule as SourceModuleSymbol;
-                return (object)sourceModuleSymbol == null ? null : sourceModuleSymbol.DeclaringCompilation;
```

Comment:
>(object) [](http://example.com/codeflow?start=23&length=8)

Why remove the cast?

Reply:
Ah good catch. I was undoing edits in this file (had temp converted it to `?.`). Apparently didn't undo far enough.